_posts/2019-11-28-jumpstart.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: JumpStart
+date: 2019-11-28
+categories:
+  - projects
+description: A display to jumpstart your day!
+image: https://www.csh.rit.edu/~mom/drno/Jumpstart.png
+image-sm: https://www.csh.rit.edu/~mom/drno/Jumpstart.png
+author: Beckett Jenen
+author-image: https://www.csh.rit.edu/~mom/drno/me.jpg
+author-bio: First year Computer Science major.
+author-email: beckettjenen@gmail.com
+author-social:
+  github: https://github.com/Dr-N0
+  linkedin: https://www.linkedin.com/in/beckettjenen/
+---
+
+# What is Jumpstart?
+Jumpstart dashboard that runs on a TV in the CSH 'Vader lobby'. Its primary goal is to display 
```

Comment:
I'd change `'Vader lobby' to just `elevator lobby`, as that's just a nickname that may not make sense to some readers.

Reply:
`Jumpstart dashboard that` -> `Jumpstart is a dashboard that`